src/dev/flang/tools/docs/Html.java

```diff
@@ -218,6 +220,52 @@ private static String source(AbstractFeature feature)
       .replace("$1", featureURL(feature));
   }
 
+  static String processComment(String name, String s)
+  {
+    var codeNo = new ArrayList<Integer>();
+    var codeLines = new ArrayList<String>();
+    var codeParsedLines = new ArrayList<String>();
+    var resultLines = new ArrayList<String>();
+
+    s.lines().forEach(l ->
+      {
+        if (l.matches("^\\s\\s\\s\\s.*"))
+          {
+            /* code comment */
+            codeLines.add(l);
+          }
+        else if (l.matches("^$"))
+          {
+            /* avoid adding lots of line breaks after code comments */
+            if (codeLines.isEmpty()) {
+              resultLines.add(l);
+            }
+          }
+        else
+          {
+            if (!codeLines.isEmpty())
+              {
+                var id = "fzdocs." + name + codeNo.size();
+                /* dump codeLines into a <code></code> block */
+                codeParsedLines.add("<div class=\"runcode-wrapper\"><i class=\"far fa-spinner fa-spin\"></i><div class=\"mb-15 runcode\" style=\"display: grid; grid-template-columns: repeat(auto-fill, minmax(min(100%,40ch), min(100%, 80ch))); max-width: 49rem; opacity: 0;\"><div class=\"position-relative\"><form id=\"" + id + "\"><textarea class=\"codeinput\" required=\"required\" maxlength=\"4096\" id=\"" + id + ".code\" name=\"code\" rows=\"3\" spellcheck=\"false\">");
```

Comment:
I find this to be hard to read. Can you use multiline strings """ or Stringbuilder or sth?